site/docs/v1/tech/samlv2/pagerduty.adoc

```diff
@@ -0,0 +1,85 @@
+---
+layout: doc
+title: SAML v2 & PagerDuty
+description: Setting up PagerDuty to login using FusionAuth via SAML v2
+navcategory: login-methods
+---
+
+== PagerDuty + FusionAuth Integration Benefits
+
+* Simplify the user login experience because each user only has to remember one set of credentails.
+* Maintain centralized control of who has access to your Pager Duty account.
+* Authorized users gain transparent access between all applications using FusionAuth as an identity provider.
+* Increase security and flexibility of login credentials, including using passkeys and multi-factor authentication
+
+== How it Works
+
+* When a user reaches the PagerDuty login screen, they can sign in using their credentials from FusionAuth.
+* User management and access is managed by FusionAuth.
+
+== Requirements
+
+*In PagerDuty:* you must have a https://www.pagerduty.com/pricing/[Business or Digital Operations plan] and be the *Account Owner* in order to enable Single Sign-On. For more information about this feature, please visit their https://support.pagerduty.com/docs/sso[SSO support page]. If you do not have this role, please reach out to the **Account Owner** within your organization to configure the integration.
+
+*In FusionAuth:* you need to be able to configure a SAML connection on an application. This can be done manually with a user account assigned either the `admin` or `application_manager` role. It can also be done with an API key that has the `application` permission.
+
+== Support
+
+If you need help with this integration, you can request assistance in one of two ways.
+
+* If you have a plan which includes technical support, please https://account.fusionauth.io/account/support/[open a support ticket].
+* If you do not have a plan which includes technical support, link:/community/forum/[please create a forum post].
+
+You can review our link:/docs/v1/tech/admin-guide/technical-support[Technical Support documentation] for more details.
+
+== Integration Walkthrough
+
+=== In PagerDuty
+
+. The _Account Owner_ can configure the SSO settings by navigating to [breadcrumb]#User Icon -> Account Settings -> Single Sign-On#.
+. Select the *SAML* radio button.
+. Copy the *SAML Endpoint URL*.
+
+_________________________________________________________________________________________________
+For more details, please visit
+https://support.pagerduty.com/docs/sso[PagerDuty SSO support page]
+_________________________________________________________________________________________________
+
+=== In FusionAuth
+
+. Click the [uielement]#Applications# link in the sidebar.
```

Comment:
So this is actually a [breadcrumb]#Applications# because it is a navigation element.